src/astro/sql/operators/agnostic_load_file.py

```diff
@@ -82,32 +76,25 @@ def load_data(
         for file in get_files(
```

Comment:
This can probably be simplified by using the `File` instance

Reply:
This is coming from files, we are using this method to resolve pattern in paths.